addons/web_editor/static/src/components/history_dialog/history_dialog.js

```diff
@@ -37,6 +38,9 @@ class HistoryDialog extends Component {
     }
 
     async init() {
+        const isDarkMode = cookie.get("color_scheme") === "dark";
+        document.documentElement.style.setProperty('--border-color', `var(--border-color-${isDarkMode ? "dark" : "normal"})`);
+        document.getElementsByClassName("modal-content")[0].style.height = "100%";
```

Comment:
HistoryDialog is using Dialog, when inspecting, i found that modal-content only have max-height and i need a fixed height. so to avoid touching modal_content, i had 2 solutions inherit it in xml and add a style or do it in js. i prefered the second one ( i don't have a valid reason, it's just preference ) what do you think ?

Reply:
I don't really understand why we need a fixed height (moreover, it is not really fixed height since you define `100%` and not a height in pixel)? How do we render it in mobile for instance?